src/components/Missions/Missions.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
- [ ] Kindly write more unit tests for the other components using React Testing Library and Jest.